fastlane/Fastfile

```diff
@@ -344,12 +345,94 @@ lane :build_test_app_and_frameworks do
   )
 end
 
+lane :xcmetrics do |options|
+  next unless is_check_required(sources: sources_matrix[:xcmetrics], force_check: @force_check)
+
+  ['test_output/', 'performance/'].each { |dir| FileUtils.remove_dir(dir, force: true) }
+
+  scan(
+    project: xcode_project,
+    scheme: 'StreamChatUITestsApp',
+    testplan: 'Performance',
+    result_bundle: true,
+    derived_data_path: derived_data_path,
+    cloned_source_packages_path: source_packages_path,
+    clean: is_localhost,
+    xcargs: buildcache_xcargs,
+    prelaunch_simulator: true
+  )
+
+  sh('xcparse logs test_output/StreamChatUITestsApp.xcresult test_output/')
+  action_log = File.read(Dir.glob('test_output/**/action.txt').first)
+  branch_performance = xcmetrics_log_parser(log: action_log)
+
+  sh("git clone git@github.com:GetStream/stream-swift-performance-benchmarks.git #{File.dirname(performance_path)}")
+  previous_release_branch = "release/#{last_git_tag}"
+  performance_benchmarks = JSON.parse(File.read(performance_path))
+  develop_performance = performance_benchmarks['develop']
+  previous_release_performance = performance_benchmarks[previous_release_branch]
+
+  github_comment = "## Performance Metrics\n| `metric` | `#{current_branch}` | `#{previous_release_branch}` | `develop` | `status` |\n| - | - | - | - | - |\n"
+  [:startup_time, :message_list_load_time, :channel_list_load_time, :message_list_scroll_time, :channel_list_scroll_time].each do |metric|
```

Comment:
Another nice to have would be to show the amount of "%" that it was better or worse 💡 Example:

| `metric` | `ci/xcmetrics` | `release/4.47.1` | `develop` | `status` | `diff` |
| - | - | - | - | - | - |
| Startup time | 1.4s | 1.5s | 42s | 🟢 | 2% ⬆️ |
| Message list load time | 2.7s | 2.2s | 2s | 🟡 | -6% ⬇️  |
| Channel list load time | 2.1s | 1s | 1s | 🔴 | -11% ⬇️  |
| Message list scroll time | 2.6s | 42s | 42s | 🟢 | 5% ⬆️ |
| Channel list scroll time | 2.2s | 42s | 42s | 🟢 | -1% ⬇️  |